### PR TITLE
#159053952 Integrate Travis CI

### PR DESCRIPTION
**Pull request by dcDalin**
#### What does this PR do?
Integrate Travis CI and add Build badge to the README.md

#### Description of tasks to be completed
Show if the project is successfully building or not through the build badge on README.md

#### How should this be manually tested?
View the Build Profile on _travis-ci.org_
> [ah-titans Travis Profile](https://travis-ci.org/andela/ah-titans)

#### Any background context you want to provide?
Build will fail as tests are not yet integrated.

#### What are the relevant Pivotal Tracker Stories?
[#159053952](https://www.pivotaltracker.com/story/show/159053952)

### CODE REVIEW

**Comment by johnmusiu** [comment]
Looks good to me

**Comment by migot01** [comment]
Looks good

File: .gitignore [hunk 1]
```diff
@@ -91,3 +91,7 @@ ENV/
 
 # SQLite3
 db.sqlite3
+
+
+# VSCode settings
+.vscode
```
Comment (ridgekimani) [inline .gitignore]: Add an end of line

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+language: python
+
+python:
+    - "3.6"
+    
+install:
+  - pip install -r requirements.txt
```
Comment (ridgekimani) [inline .travis.yml]: Add an end of line here.

**Comment by ridgekimani** [comment]
How sure are we that travis will work? 
I think you should more research on how to integrate travis with a django project because there are things which haven't been considered here; like running migrations, creating a test database, setting environment variables like the secret key and therefore this might not work.

**Comment by janetnim** [comment]
Also maybe test validation on the user details like what should happen when they input a password of less than 8 characters etc

**Comment by johnmusiu** [comment]
@janetnim This PR doesn't add validations, only configures Travis CI.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    
+install:
+    - pip install -r requirements.txt
+
+before_script:
+    - psql -c "CREATE DATABASE test;" -U postgres
+
+script:
+    - python manage.py test
+
```
Comment (ridgekimani) [inline .travis.yml]: End of line

Comment (dcDalin) [reply .travis.yml]: I don't exactly understand what you mean by *End of line*, kindly elaborate more on that.

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+before_script:
+    - psql -c "CREATE DATABASE test;" -U postgres
+    - export SECRET_KEY="this_is_the_secret_key"
+
+script:
+    - coverage run --source=. manage.py test
+
+after_success:
+    - coveralls
```
Comment (ridgekimani) [inline .travis.yml]: End of line

**Review by ronaldndirangu — APPROVED** [review]
Everything seems to be working okay after fixing the bug, the tests are passing.

**Review by petermwash — APPROVED** [review]
Everything looks fine to me.

**Pull request merged**